crates/apis/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+//! JS APIs for Javy.
+//!
+//! This crate provides JS APIs you can add to Javy.
+//!
+//! Example usage:
+//! ```
+//! # use anyhow::{anyhow, Error, Result};
+//! use javy::{quickjs::JSValue, Runtime};
+//! use javy_apis::RuntimeExt;
+//!
+//! let runtime = Runtime::new_with_defaults()?;
+//! let context = runtime.context();
+//! context.global_object()?.set_property(
+//!    "print",
+//!    context.wrap_callback(move |_ctx, _this, args| {
+//!        let str = args
+//!            .first()
+//!            .ok_or(anyhow!("Need to pass an argument"))?
+//!            .to_string();
+//!        println!("{str}");
+//!        Ok(JSValue::Undefined)
+//!    })?,
+//! )?;
+//! context.eval_global("hello.js", "print('hello!');")?;
+//! # Ok::<(), Error>(())
+//! ```
+
+#![deny(
```

Comment:
I want to add this at the crate level to trigger failures for certain issues with docs.

We could do this in the Makefile but that means just running a plain `cargo docs` or `cargo check` won't cause them to raise. `missing_docs` being in here should also cause the Rust Analyzer to indicate an error when there are missing docs. I am open to moving it to the Makefile though.

Reply:
I'm also open to removing this change from this PR and figuring out an approach we can use consistently across our crates. But I'd like to eventually have CI fail if a public API on the crates we're publishing do not include docs.

Reply:
I'd be in favor of removing this change and then figuring out a way to consistently handle docs across crates. My point of view is that we'd need to figure out a consistent way to do this anyway given that we want to add documentation to Javy as whole.